roles/vault/tasks/bootstrap.yml

```diff
@@ -25,7 +25,7 @@
   changed_when: no
   when: "'Vault is already initialized' not in vault_init_output.stdout"
   set_fact:
-    vault_token: "{{ (vault_init_output.stdout|from_json)['root_token'] }}"
+    vault_root_token: "{{ (vault_init_output.stdout|from_json)['root_token'] }}"
```

Comment:
Hmm, that kind of creates an issue for us when we want to use the `vault_token` in other parts of our setup, because we can't know whether we need `vault_token` (if Vault were just bootstrapped) or `vault_root_token` (if it hasn't been). Do you have any ideas? We could use `{{ vault_token | defualt(vault_root_token) }}` (or the reverse). 

Alternately, we could resolve the ambiguity beforehand and make the following task:
```yaml
- set_fact: 
    vault_token: "{{ vault_root_token }}"
  when: "{{ vault_root_token is defined }}"
```